lib/train/transports/kubernetes.rb

```diff
@@ -0,0 +1,72 @@
+# encoding: utf-8
+#
+# Copyright 2017, Noah Kantrowitz
```

Comment:
Please change to author

Reply:
Copyright is correct as I remain the owner of this code. Unless you're requesting I assign copyright to Chef Software, which would be unusual?